### ISSUE TITLE
Why are we using the dev version of wysiwyg module?

### ISSUE THREAD

**Issue by shrop** [hb5co/hb5co-drupal#20]
@bfodeke why are we using the dev version? Need newer fixes and features?

Looks like a new dev release is available if we need to upgrade:
Currently at 7.x-2.2+70-dev (2015-Oct-24) and the new version is 7.x-2.x-dev (2015-Nov-21)

**Comment by bfodeke** [comment]
Yes, new features. The dev version is the only one that supports ckeditor 4

## 

Bayo Fodeke
Sent with Airmail

**Comment by hb5co** [comment]
@bfodeke thanks man! ok, looks like we could update this and test :)

**Comment by shrop** [comment]
@bfodeke thanks man! ok, looks like we could update this and test :)
